resources/public/dashboard/index.html

```diff
@@ -257,11 +257,6 @@
        url: "/metrics/v1/mbeans/puppetlabs.puppetdb.storage:name=gc-time",
        format: d3.format(",.3s"),
        snag: getPercentileSnag},
-      {description: "DLO Compression",
```

Comment:
I think @rbrw can comment more on this, but we found that DLO as it is written, wasn't "connected up", so it was impossible to hit that code, other than in tests. We made the decision that if we've gone this far without it, no need to reimplement it with the new DLO code.